### DISCUSSION TITLE
Auto-refresh task logs in UI

### DISCUSSION THREAD

**Discussion by aamster** [apache/airflow#34061] [General]
### Description

It would be nice if task logs auto refreshed in the UI so that I don't need to continually refresh the page to see log updates. 

### Use case/motivation

Monitoring logs in real-time

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Aren't they ? In which case? Can you be specific which case of the logs you are talking about (and which version of Airflow you are looking at. Screenshots would be nice to explain

**Comment by aamster** [comment]
No, they are not. Using 2.5.1. 

![Screen Shot 2023-01-30 at 1 44 28 PM](https://user-images.githubusercontent.com/5454341/215567043-ad0a228c-b18e-46b9-83fc-447c5f64b05e.png)

**Comment by potiuk** [comment]
Did you try the logs in Grid view? That's why I am asking what logs are you talking about and asked for screenshots

**Comment by potiuk** [comment]
This was a request for precise description of what you are asking for.

**Comment by potiuk** [comment]
(and yes auto-refresh of log page is not yet implemented - as opposite to the `logs` tab in the grid view)

**Comment by jscheffl** [comment]
Mhm, just found this (open?) issue. We have LogTailing active and are happy, since 2.5.0 this has been added. But it dependds a bit on the backend of log adapter if this is supported. It _should_ already work with standard file based logging and I can confirm also in Celery sourcing from a running worker this is possible. That was my first PR in contribution here:

https://github.com/apache/airflow/pull/26169

Can you (@aamster) describe which log backend you are using and missing this feature? Because in general this is available since 2.5.0

**Comment by potiuk** [comment]
Let's convert it into discussion - this issue was raised some time ago - in January so this is likely old version of the Ui.
